kieserver/modules/kieserver/module.yaml

```diff
@@ -28,17 +27,17 @@ envs:
 # remember to also update "JBPM_WB_KIE_SERVER_BACKEND_JAR" value
 artifacts:
 - name: "rhpam_kie_server_distribution.zip"
-  # rhpam-7.13.4-kie-server-ee8.zip
+  # rhpam-7.13.5-kie-server-ee8.zip
   md5: "adb6f8394caa54c444cf0571d2a8debb"
 - name: "rhpam_business_central_distribution.zip"
-  # rhpam-7.13.4-business-central-eap7-deployable.zip
+  # rhpam-7.13.5-business-central-eap7-deployable.zip
   md5: "87c555d71825b88d3576fe6fba3a998c"
 - name: "slf4j-simple.jar"
   # slf4j-simple-1.7.30.redhat-00005.jar
   md5: "44e25fa5b93454bc9cfc005f8cb98b53"
-- name: "kie-server-services-jbpm-cluster-7.67.0.Final-redhat-00024.jar"
+- name: "kie-server-services-jbpm-cluster-7.13.5.Final-redhat-00024.jar"
```

Comment:
@spolti changes in jbpm-event-emitters-kafka and kie-server-services-jbpm-cluster versions seems wrong

Reply:
it is the script version the regex catches this as well.